python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "ApdbSchema"
 "@id": "#apdbSchema"
-version: "7.0.1"
+version: "7.0.2"
```

Comment:
Metadata changes (like units) do not need version update as the database schema does not depend on these changes. Each version increment needs a new migration script in dax_apdb_migrate, I'd prefer to avoid adding scripts that do nothing.